src/utils/formatting.ts

```diff
@@ -3,12 +3,11 @@ import { BetId } from "../types/entities";
 
 export const formatToFourDecimals = (amount: string) => {
   const parsedAmount = parseFloat(amount);
-  if (parsedAmount === 0) return "0";
+  if (parsedAmount === 0) return "0.0000";
```

Comment:
if having `0` return as `0.0000` is intended we dont need this line at all

Reply:
I’ll test via the unit tests

Reply:
Its still a valid condition.

Reply:
i see -- its because we have a condition for if the value is less than `0.0001`, but we also want `0` to return a `toFixed(4)` value -- sneaky